cmd/virt-handler/virt-handler.go

```diff
@@ -284,10 +284,6 @@ func (app *virtHandlerApp) Run() {
 		glog.Fatalf("Error preparing the certificate manager: %v", err)
 	}
 
-	if err := app.setupTLS(factory); err != nil {
```

Comment:
Inside `app.setupTLS` function we set the `serverTLSConfig` through  `webhooks.SetupTLSForVirtHandlerServer` https://github.com/fossedihelm/kubevirt/blob/53a18323574d12c394f978f98ac7a565a9a39deb/cmd/virt-handler/virt-handler.go#L589
The latter uses the app.clusterConfig` which is initalized only in line 296 https://github.com/kubevirt/kubevirt/blob/4659ca8489748cd50f32bbaa3107ff8b46ef4548/cmd/virt-handler/virt-handler.go#L296. Without this move it will result `nil` and the setupTLSConfig will not work properly.
Thanks